src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationInMultiblockStructuresOne.java

```diff
@@ -0,0 +1,118 @@
+/*
+CommentsIndentation
+tokens = (default)SINGLE_LINE_COMMENT, BLOCK_COMMENT_BEGIN
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+public class InputCommentsIndentationInMultiblockStructuresOne {
```

Comment:
I don't see usage of new Inputs.

Reply:
Extremely sorry for that..
I am actually working on three issues parallely and misplaced the files in a different branch.